feats/feats.yml

```diff
@@ -839,19 +842,19 @@
   description: |
     Following a trail is an everyday part of your life, and as such it is no more taxing than breathing or blinking. Whether you are an experienced woodsman, a bounty hunter for the Galactic Senate, or a humanoid cat creature with heightened senses, your expertise at tracking allows you to get it done faster than the average hunter.
   effect: |
-    You can move at your full speed while tracking your quarry rather than the reduced speed typically incurred.
+    You move unhindered and at full speed when tracking your quarry.
 - !
   name: Ferocious Minions (I - III)
   prerequisites:
     tier1:
       Attribute:
-      - Influence: 3
+      - Invoking Attribute: 4
     tier2:
```

Comment:
This was print-only. It was messy, from a .yml standpoint, to have a prereq that was not specific.

Reply:
@jcte02 I was all set to merge this but @ucffool is right... gotta list 'em all out here. 

I'm neck deep in getting this PDF to the printer, so I can't check right now, but something like `Creation, Entropy, Influence or Logic X` per your comment.